client/src/proxies/proxy.js

```diff
@@ -11,7 +11,7 @@ export default class BaseProxy {
 
     try {
       const response = await axios.get(requestUrl);
-      return response.data;
+      return response;
```

Comment:
why did you remove the response.data , 
do you expect to anything else? 
if requested done successfully (it's mean status 200) do return his data 
unless you going to catch section 
then expected to parse the error , like: 
`error.response.data.error.message`